spec/factories/tenants.rb

```diff
@@ -1,6 +1,6 @@
 FactoryBot.define do
   factory :tenant do
-    sequence(:external_tenant) { |n| n + rand(1000) }
+    sequence(:external_tenant)
```

Comment:
Yes, this seems like the proper fix to me.

It looks like rubocop helped us get to the previous version. Commit https://github.com/ManageIQ/catalog-api/pull/101/commits/2ac35f25d16b3449acc8657b7eedae5708a0ed5a

Reply:
thanks rubocop! :laughing:

Reply:
Thanks @eclarizio for finding it!